test/middleware/spec.back-link.js

```diff
@@ -81,6 +81,8 @@ describe('Back Links', function () {
     });
 
     it('adds the previous step to res.locals.backLink', function () {
+        req.get.withArgs('referrer').returns('http://example.com/referrer');
+        req.baseUrl = '/referrer';
```

Comment:
I mean that with 'http://example.com/referrer/step2' as a referrer is makes the assumption in the test more specific.